models/path.py

```diff
@@ -115,6 +115,15 @@ def set_controller(cls, controller=None):
     @staticmethod
     def get_paths(circuit, max_paths=2, **kwargs):
         """Get a valid path for the circuit from the Pathfinder."""
+        # pylint: disable=fixme
+        # XXX: temporary workaround for kytos-ng/pathfinder#37
+        if (
+            len(circuit.secondary_constraints.get('undesired_links', [])) > 0
+            or len(circuit.secondary_constraints.get('desired_links', [])) > 0
+            or len(circuit.primary_constraints.get('undesired_links', [])) > 0
+            or len(circuit.primary_constraints.get('desired_links', [])) > 0
+        ):
+            max_paths += 10
```

Comment:
This should be handled by the Pathfinder Napp. The number 10 is a magical value that may work for most cases, but it depends on the topology and the constraints provided by the user. Since this is a workaround while kytos-ng/pathfinder#37 is not fixed, I believe this may be an acceptable solution.